### PR TITLE
feat(llm): support multiple property_type & importing graph from the entire doc

### PR DESCRIPTION
**Pull request by vichayturen**
Done:
1. Fixed bug of config qianfan llm api
2. Added two enums for properties settings
3. support different property type
4. support list/set cardinality for property_key

TODO:
- [ ] enhance the logic for split chunk for vector & graph (maybe we should divide them separately)

### CODE REVIEW

File: hugegraph-llm/src/hugegraph_llm/operators/hugegraph_op/schema_manager.py [hunk 1]
```diff
@@ -37,20 +37,20 @@ def run(self, context: Optional[Dict[str, Any]]) -> Dict[str, Any]:
...
+        #     vertices.append(vertex)
+        # edges = []
+        # for el in schema["edgelabels"]:
+        #     edge = {
+        #         "edge_label": el["name"],
+        #         "source_vertex_label": el["source_label"],
+        #         "target_vertex_label": el["target_label"],
+        #         "properties": el["properties"],
+        #     }
+        #     edges.append(edge)
```
Comment (imbajin) [inline hugegraph-llm/src/hugegraph_llm/operators/hugegraph_op/schema_manager.py:52]: need keep it?

File: hugegraph-llm/src/hugegraph_llm/utils/graph_index_utils.py [hunk 2]
```diff
@@ -113,8 +110,9 @@ def import_graph_data(data: str, schema: str) -> Union[str, Dict[str, Any]]:
 
         context = builder.commit_to_hugegraph().run(data_json)
         gr.Info("Import graph data successfully!")
+        print(context)
```
Comment (imbajin) [inline hugegraph-llm/src/hugegraph_llm/utils/graph_index_utils.py:113]: Clean it?

File: hugegraph-llm/src/hugegraph_llm/operators/hugegraph_op/commit_to_hugegraph.py [hunk 3]
```diff
@@ -112,17 +142,21 @@ def load_into_graph(self, vertices, edges, schema):
...
+
         for vertex in vertices:
             vertex_label = vertex["name"]
             properties = vertex["properties"]
             nullable_keys = vertex["nullable_keys"]
             primary_keys = vertex["primary_keys"]
-            for prop in properties:
-                self.schema.propertyKey(prop).asText().ifNotExist().create()
+            # for prop in properties:
+            #     self.schema.propertyKey(prop).asText().ifNotExist().create()
```
Comment (simon824) [inline hugegraph-llm/src/hugegraph_llm/operators/hugegraph_op/commit_to_hugegraph.py:162]: remove directly?

File: hugegraph-llm/src/hugegraph_llm/operators/hugegraph_op/commit_to_hugegraph.py [hunk 4]
```diff
@@ -132,8 +166,8 @@ def init_schema_if_need(self, schema: object):
             source_vertex_label = edge["source_label"]
             target_vertex_label = edge["target_label"]
             properties = edge["properties"]
-            for prop in properties:
-                self.schema.propertyKey(prop).asText().ifNotExist().create()
+            # for prop in properties:
+            #     self.schema.propertyKey(prop).asText().ifNotExist().create()
```
Comment (simon824) [inline hugegraph-llm/src/hugegraph_llm/operators/hugegraph_op/commit_to_hugegraph.py:173]: ditto

File: hugegraph-llm/src/hugegraph_llm/operators/kg_construction_task.py [hunk 5]
```diff
@@ -60,16 +59,17 @@ def fetch_graph_data(self):
         return self
 
     def chunk_split(
-            self,
-            text: Union[str, List[str]],  # text to be split
-            split_type: Literal["paragraph", "sentence"] = "paragraph",
-            language: Literal["zh", "en"] = "zh"
+        self,
+        text: Union[str, List[str]],  # text to be split
+        split_type: Literal["document", "paragraph", "sentence"] = "document",
```
Comment (simon824) [inline hugegraph-llm/src/hugegraph_llm/operators/kg_construction_task.py:64]: Why is the default value changed to `document`, does it have a better effect?

File: hugegraph-llm/src/hugegraph_llm/utils/graph_index_utils.py [hunk 6]
```diff
@@ -113,8 +110,9 @@ def import_graph_data(data: str, schema: str) -> Union[str, Dict[str, Any]]:
...
         return json.dumps(context, ensure_ascii=False, indent=2)
-    except Exception as e: # pylint: disable=W0718
+    except Exception as e:  # pylint: disable=W0718
```
Comment (simon824) [inline hugegraph-llm/src/hugegraph_llm/utils/graph_index_utils.py:115]: Is this configuration universal? Does it need to be added to `pylint.conf`?

File: hugegraph-llm/src/hugegraph_llm/models/llms/openai.py [hunk 7]
```diff
@@ -141,7 +140,7 @@ def num_tokens_from_string(self, string: str) -> int:
     def max_allowed_token_length(self) -> int:
         """Get max-allowed token length"""
         # TODO: list all models and their max tokens from api
-        return 2049
+        return 8192
```
Comment (simon824) [inline hugegraph-llm/src/hugegraph_llm/models/llms/openai.py:143]: why increase to 8192？

Comment (imbajin) [reply hugegraph-llm/src/hugegraph_llm/models/llms/openai.py:143]: > why increase to 8192？

`2049` is outdated (in 2023y)

BTW, this method is not used now (may remove it later)

File: hugegraph-llm/src/hugegraph_llm/operators/kg_construction_task.py (hunk 5 above)
Comment (imbajin) [reply hugegraph-llm/src/hugegraph_llm/operators/kg_construction_task.py:64]: > Why is the default value changed to `document`, does it have a better effect?

handle it in next PR (we need refactor the split logic for better effect)

**Review by imbajin — APPROVED** [review]
Merge it to avoid blocking other PRs (handle chunk split logic in another PR later) @vichayturen

**Pull request merged**